tests/phpunit/tests/test-accept-languages-collection.php

```diff
@@ -21,10 +32,19 @@ public static function set_up_before_class() {
 	 * @return PLL_Language
 	 */
 	protected function get_known_language( $locale ) {
-		$language = self::$known_languages[ $locale ];
-		$language['slug'] = $language['code'];
-		$language['w3c']  = isset( $language['w3c'] ) ? $language['w3c'] : str_replace( '_', '-', $language['locale'] );
-		return new PLL_Language( $language );
+		$language               = self::$known_languages[ $locale ];
+		$language['locale']     = $locale;
+		$language['slug']       = $language['code'];
+		$language['w3c']        = isset( $language['w3c'] ) ? $language['w3c'] : str_replace( '_', '-', $language['locale'] );
+		$language['rtl']        = 'rtl' === $language['dir'] ? 1 : 0;
+		$language['term_group'] = 0;
+		$result = self::$model->add_language( $language );
+
+		$this->assertNotInstanceOf( WP_Error::class, $result, "{$locale} language is not created." );
+
+		self::$model->clean_languages_cache();
+
+		return self::$model->get_language( $language['slug'] );
```

Comment:
Since your code now dependent of `PLL_Model`, why not just creating the language the usual way with our `create_language()` helper method?

Reply:
Good idea! Don't know why I didn't thought about it 😞 
Done in 2df8e11